packages/react-composites/src/composites/common/utils.tsx

```diff
@@ -53,3 +53,12 @@ export const useMinMaxDragPosition = (modalLayerHostId: string, rtl?: boolean):
  * @private
  */
 export const defaultSpokenLanguage = 'en-us';
+
+/**
```

Comment:
Move this to `acs-ui-common` and just have one instance to use throughout all packlets